app/src/main/java/io/appium/uiautomator2/model/UiObjectElement.java

```diff
@@ -194,60 +189,9 @@ public AccessibleUiObject getChild(final Object selector) throws UiObjectNotFoun
                 throw new ElementNotFoundException();
             }
             List<UiObject2> children = ((UiObject2) root.getValue()).findObjects((BySelector) selector);
-            return toAccessibleUiObjects(children);
-        }
-        return this.getChildElements((UiSelector) selector);
-    }
-
-    private ArrayList<AccessibleUiObject> getChildElements(final UiSelector sel) throws UiObjectNotFoundException {
```

Comment:
ah, so is the main change `ByUiAutomatorFinder.matchDescendantElements` instead of here...?
I see.

Reply:
not really. This was just a part of refactoring. The performance fix was to set a proper timeout